apps/taxes/src/modules/channel-configuration/channel-configuration-merger.ts

```diff
@@ -0,0 +1,18 @@
+import { ChannelFragment } from "../../../generated/graphql";
+import { ChannelsConfig } from "./channel-config";
+
+export class ChannelConfigurationMerger {
```

Comment:
Way back, I refactored such functions to one-method classes after feedback from @lkostrowski. Since then, I have been sticking to object-oriented design, at least around the providers. Nothing wrong with good ol' functions, but since most of the code benefits from being a class, I didn't want to break that convention.

Reply:
If you feel it helps you with keeping the consistency then okay, carry on :) 

Not a big issue to refactor it in the future if we decide to modify the approach.

Reply:
Its not really OOP vs FP when we discuss if function is wrapped with an object or not. In FP there are classes too.

Its only a matter of providing the context. Its really hard to understand hundreds of loose functions like `mapData` or `transformOrder` when everything is exported without context.

On the other hand you can have `OrderTransformer.fromXtoY` which is explicit and understandable.

Of course you can export loose function `transformOrderFromYtoX` but this is just more obscure

Also it doesn't have to be class. It can be 

`const ChannelConfigurationMerger = { merge }`

Everything works as long as we don't confuse what does what. 

In Java we would use namespaces that solve this problem. In TS we need to emulate this